src/runtime/pkg/katautils/config.go

```diff
@@ -400,6 +402,20 @@ func (h hypervisor) defaultMemOffset() uint64 {
 	return offset
 }
 
+func (h hypervisor) defaultMaxMemSz() uint64 {
```

Comment:
Nit: I'd consider adding the units for the maximum memory size to the name of this method and `hypervisor.DefaultMaxMemorySize` to avoid any confusion.